source/frontend/src/features/documents/list/DocumentResults/DocumentResultsColumns.tsx

```diff
@@ -94,6 +80,25 @@ const renderFileName = (onViewDetails: (values: Api_Document) => void) => {
   };
 };
 
+//export const DateCell = ({ cell: { value } }: CellProps<Date, string>) => prettyFormatDate(value);
+
+function renderUploaded(cell: CellProps<Api_Document, string>) {
+  return (
+    <Row className="no-gutters">
+      <Col>{prettyFormatDate(cell.row.original.appCreateTimestamp)}</Col>
+      <Col xs="auto">
+        <StyledIcon>
+          <TooltipIcon
+            toolTipId="initiator-tooltip"
+            toolTip={cell.row.original.appCreateUserid}
```

Comment:
I'd think this should be the last updated not the creator, but good to check requirements here.

Reply:
I think is the creator given that the uploaded column is showing the date of creation